### PR TITLE
change super compile so pretty output is default 

### PR DESCRIPTION
**Pull request by mccanne**

### CODE REVIEW

File: cli/outputflags/flags.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ import (
 type Flags struct {
 	anyio.WriterOpts
 	DefaultFormat string
+	DefaultPretty bool
```
Comment (nwt) [inline cli/outputflags/flags.go:25]: Nit: Call this DefaultZSONPretty since both JSON and ZSON can be pretty?

**Review by nwt — APPROVED** [review]

File: cli/outputflags/flags.go (hunk 1 above)
Comment (mccanne) [reply cli/outputflags/flags.go:25]: Hmm, good point.  I think the logic is off.  I'll look a little closer and write some tests.

Comment (mccanne) [reply cli/outputflags/flags.go:25]: Here's the problem...
```
; super compile -f json 'from x'
cannot use -z or -Z with -f
```

**Comment by mccanne** [comment]
I don't see any easy way to do what I wanted without messing up the generic output flags logic.  We probably need to rethink the logic to do what I want.  I'll live with typing -Z for now...

**Pull request closed without merging**